### ISSUE TITLE
non_key_attributes support?

### ISSUE THREAD

**Issue by p0fi** [cloudposse/terraform-aws-dynamodb#14]
It seems this module does not support the `non_key_attributes` argument of the `aws_dynamodb_table` resource. Any chance this gets supported as an input variable?

**Comment by aknysh** [comment]
@p0fi  thanks for testing.
We'd like to support that.
Do you have something working already so you could submit a PR?

**Comment by p0fi** [comment]
Im fairly new to terraform so I have not a lot of experience with it. I tried to implement it tho, but it looks like terraform has its problems with nested lists in variables. I couldn't get it to work. To bypass the nested lists issue I thought it might be a good idea to pass it as a comma separated string in the `global_secondary_index_map` variable and than before adding it to the `aws_dynamodb_table` reconstruct it as a list but I failed so far 😞
